doc/Quickstart.md

```diff
@@ -19,10 +19,12 @@ brew services start postgresql
 
 **Note:** The command `brew services start postgresql` starts the service of PostgreSQL. If later `./bin/setup` aborts, make sure that the `PostgreSQL` service is running. Verify with `brew services list` that has a status `started` and looking green. If the status is `started` but coloured orange, fix the errors indicated in the log located in `/usr/local/var/log/postgres.log`
 
-## Run PostgreSQL on Fedora 34
+## Run PostgreSQL on Fedora 42
+
+Better use instructions for container below because with Fedora ou can't controll PostgreSQL version.
```

Comment:
```suggestion
Better use instructions for container below because with Fedora you can't control PostgreSQL version.
```

Reply:
:angry: that `y` key! I submitted a support request, haha